packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -147,17 +148,32 @@ function PreviewView() {
 
   function onMouseDown(e: MouseEvent<HTMLDivElement>) {
     e.preventDefault();
+    const elementName = (e.target as Element).className;
```

Comment:
I missed that in #551 we added this extra layer of capturing mouse events. What is the reason for it. The pressing state is not read in this component except from passing it down to Preview

Reply:
In #551, the pressing state was moved to previewView so it remains true when the mouse leaves and reenters device screen during multitouch gesture. Before this change if mouse reentered multitouch wouldn't send touches unless mouse button was released and clicked again within the phone frame. However, I can revert this if necessary.